src/Pos/Explorer/Txp/Toil/Logic.hs

```diff
@@ -132,3 +150,59 @@ getTxRelatedAddrs (UnsafeTx {..}, _, _) undo =
     toSet = HS.fromList . toList
     -- Safe here, because union of non-empty sets can't be empty.
     unionNE lhs rhs = NE.fromList $ toList $ HS.union (toSet lhs) (toSet rhs)
+
+combineBalanceUpdates :: BalanceUpdate -> [(Address, (Sign, Coin))]
```

Comment:
Why not store `HashMap Address (Sign, Coin)` instead of `[(Address, Coin)]` in `BalanceUpdate`?